SECURITY.md

```diff
@@ -12,3 +13,62 @@
 Please report any vulnerabilities via our [Bug Bounty Program](https://bugcrowd.com/owaspzap).
 
 Remote Code Execution for this program will be rewarded at __$1000__
+
+## Reporting a Bug
+
+Please report any bugs via our issue [Bug Issue Tracker](https://github.com/zaproxy/zaproxy/issues/new/choose)
+
+## Security Advisories
+
+Currently, there are not any published security advisories.
+
+## Scan Policy
+
+ZAP supports both active and passive scanning rules.
+
+All rules are contained in add-ons so that they can be updated quickly and easily.
+
+Active scanning is an attack on those targets.
+You should NOT use it on web applications that you do not own.
```

Comment:
The note below is more accurate, you don’t have to own it to be authorized to asses something